python/fledge/services/core/api/health.py

```diff
@@ -21,11 +21,116 @@
     | GET            | /fledge/health/storage               |
+    | GET            | /fledge/health/logging               |
     ----------------------------------------------------------
 """
 _LOGGER = logger.setup(__name__, level=logging.INFO)
 
 
+async def get_disk_usage(given_dir):
+    """
+       Helper function that calculates used, available, usage(in %) for a given directory in file system.
+       Returns a tuple of used(in KB's integer), available(in KB's integer), usage(in %)
+    """
+    disk_check_process = await asyncio.create_subprocess_shell('df -k ' + given_dir,
+                                                               stdout=asyncio.subprocess.PIPE,
+                                                               stderr=asyncio.subprocess.PIPE)
+
+    stdout, stderr = await disk_check_process.communicate()
+    if disk_check_process.returncode != 0:
+        stderr = stderr.decode("utf-8")
+        msg = "Failed to get disk stats! {}".format(str(stderr))
+        _LOGGER.error(msg)
+        raise web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
+
+    # Following output is parsed.
+    """
+        Filesystem     1K-blocks     Used Available Use% Mounted on
+        /dev/sda5      122473072 95449760  20755872  83% /
+
+    """
+    disk_stats = stdout.decode("utf-8")
+    required_stats = disk_stats.split('\n')[1].split()
+    used = int(required_stats[2])
+    available = int(required_stats[3])
+    usage = int(required_stats[4].replace("%", ''))
+
+    return used, available, usage
+
+
+async def get_logging_health(request: web.Request) -> web.Response:
+    """
+     Return the health of logging.
+    Args:
+       request: None
+
+    Returns:
+           Return the health of logging.
+           Sample Response :
+
+           {
+              "disk": {
+                "usage": 63,
+                "used": 42936800,
+                "available": 25229400
+              },
+              "levels": [
+                {
+                    "name" : "Sine",
+                    "level" : "info"
+                },
+                {
+                    "name" : "OMF",
+                    "level" : "debug"
+                }
+              ]
+           }
+
+    :Example:
+           curl -X GET http://localhost:8081/fledge/health/logging
+    """
+    response = {}
+    try:
+        from fledge.services.core.api import service as serv_api
+        from fledge.common.configuration_manager import ConfigurationManager
+        from fledge.services.core import connect
+
+        services_info = serv_api.get_service_records()
+        log_levels = []
+        excluded_services = ["Core"]
+        cf_mgr = ConfigurationManager(connect.get_storage_async())
+        for s in services_info['services']:
+            if s['type'] not in excluded_services:
+                service_name = s["name"]
+                cat_name = service_name + "Advanced"
+                if s['type'] == "Storage":
```

Comment:
Could you also check for notification & bucket advanced category?
key? I believe we have random things.